compiler/one-cmds/one-infer

```diff
@@ -0,0 +1,215 @@
+    infer_usage = 'one-infer [-h] [-v] [-C CONFIG] [-d DRIVER] [-b BACKEND] [--] [COMMANDS FOR BACKEND DRIVER]'
+    parser = argparse.ArgumentParser(
+        description='command line tool to infer model', usage=infer_usage)
+
+    _utils._add_default_arg(parser)
+
+    # TODO: add tflite/onnx-infer driver to helper message when it is implemented
+    driver_help_message = 'backend inference driver name to execute'
+    parser.add_argument('-d', '--driver', type=str, help=driver_help_message)
+
+    # get backend list in the directory
+    backends_name = [ntpath.basename(f) for f in backends_list]
+    if not backends_name:
+        backends_name_message = '(There is no available backend drivers)'
+    else:
+        backends_name_message = '(available backend drivers: ' + ', '.join(
+            backends_name) + ')'
+    backend_help_message = 'backend name to use ' + backends_name_message
+    parser.add_argument('-b', '--backend', type=str, help=backend_help_message)
+
+    return parser
+
+
+def _verify_arg(parser, args):
+    """verify given arguments"""
+    # `-d/--driver` and `-b/--backend` are mutually exclusive arguments.
+    if _utils._is_valid_attr(args, 'driver') and _utils._is_valid_attr(args, 'backend'):
+        parser.error('-d and -b options are mutually exclusive. Please use only one of them.')
+
+    missing = []
+    if not _utils._is_valid_attr(args, 'driver') and not _utils._is_valid_attr(args, 'backend'):
+        missing.append('{-d/--driver | -b/--backend}')
+    if len(missing):
+        parser.error('the following arguments are required: ' + ' '.join(missing))
+
+
+def _parse_arg(parser):
+    infer_args = []
+    backend_args = []
+    argv = copy.deepcopy(sys.argv)
+    # delete file name
+    del argv[0]
+    # split by '--'
+    args = [list(y) for x, y in itertools.groupby(argv, lambda z: z == '--') if not x]
+    
+    # one-infer [-h] [-v] [-C CONFIG] [-d DRIVER] [-b BACKEND] -- [COMMANDS FOR BACKEND DRIVER]
+    if len(args):
+        infer_args = args[0]
+        infer_args = parser.parse_args(infer_args)
+        backend_args = backend_args if len(args) < 2 else args[1]
+    # print version
+    if len(args) and infer_args.version:
+        _utils._print_version_and_exit(__file__)
+
+    return infer_args, backend_args
+
+
+def _get_executable(args, backends_list):
+    executable = None
+    missing = []
+
+    driver = getattr(args, 'driver')
+    if driver:
+        executable = _search_backend_driver(driver)
+        if executable:
+            return executable
+        missing.append(driver)
+
+    if getattr(args, 'backend'):
+        backend_base = getattr(args, 'backend') + '-infer'
+        for cand in backends_list:
+            if ntpath.basename(cand) == backend_base:
+                executable = cand
+                return executable
+        missing.append(backend_base)
+
+    if not executable:
+        raise FileNotFoundError(' or '.join(missing) + ' not found')
```

Comment:
```suggestion
    executable = None

    if _utils._is_valid_attr(args, 'driver'):
        executable = _search_backend_driver(getattr(args, 'driver'))

    if _utils._is_valid_attr(args, 'backend'):
        backend_base = getattr(args, 'backend') + '-infer'
        for cand in backends_list:
            if ntpath.basename(cand) == backend_base:
                executable = cand
                break

    if not executable:
        raise FileNotFoundError(' or '.join(missing) + ' not found')
```

1. AFAIK, just calling `getattr` may cause an error if `args` doesn't have `driver` attribute. Is it okay?
2. They've become mutually exclusive option. So, `missing` variable seems not needed anymore.

Reply:
@mhs4670go , if `driver` has higher priority (as I know) is your suggestion work as expected? or maybe my understanding is incorrect?

Reply:
@seanshpark No, above suggestion is based on the codes where both options are mutually exclusive.

Reply:
I've applied the suggested changes :D